src/open_r1/configs.py

```diff
@@ -105,6 +105,10 @@ class GRPOScriptArguments(trl.ScriptArguments):
             Maximum length for cosine scaling.
         code_language (`str`):
             Language for code format reward.
+        max_completion_len (`int`):
+            Maximum number of characters in completion.
```

Comment:
```suggestion
            Maximum number of characters in completion beyond which a -1 reward is given for soft overlong punishment.
```

Reply:
Btw it's not very clear from the variable name that it relates to the soft overlong punishment.
When running the script they'll have these two args which are very close but are for completely different purpose: `max_completion_length` and `max_completion_len`